angular/projects/admin-nrpti/src/app/records/court-convictions/court-conviction-detail/court-conviction-detail.component.html

```diff
@@ -0,0 +1,97 @@
+<main class="detail-page">
+  <div>
+    <section class="content-section">
+      <div class="content-grid">
+        <div class="grid-section-header">
+          <div class="grid-section-header-title">
+            <h4 class="grid-section-header-title-text blue-header">Shared Data [Master]</h4>
+            <span *ngIf="data && data._master.dateUpdated" class="grid-section-header-title-sub-text grey-sub-text">
+              Last Edited {{ (data && data._master.dateUpdated | date: 'mediumDate') || '-' }}
+            </span>
+          </div>
+          <div class="grid-section-header-button">
+            <button class="button-edit" (click)="navigateToEditPage()">
+              <i class="material-icons">edit</i>
+              <span>Edit</span>
+            </button>
+          </div>
+        </div>
+        <div class="grid-section-sub-header">
+          <span class="grid-section-sub-header-text">Basic Information</span>
+        </div>
+        <div class="grid-section section-1">
+          <div class="grid-item__row">
+            <span class="grid-item-name">Record Name</span>
+            <span class="grid-item-value__col">{{ (data && data._master.recordName) || '-' }}</span>
```

Comment:
These and other areas, we should be referencing from those pages that are label based, not span.